docs/tutorials/09_properties.ipynb

```diff
@@ -0,0 +1,219 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.problems.properties_container.PropertiesContainer object at 0x14c01089daf0>\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.problems import BaseProblem\n",
+    "\n",
+    "dummy_hamiltonian = None\n",
+    "\n",
+    "base_problem = BaseProblem(dummy_hamiltonian)\n",
+    "print(base_problem.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d76c0f7",
+   "metadata": {},
+   "source": [
+    "You can now modify the contents of the `PropertiesContainer` to your desire:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "fb1d2956",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AngularMomentum is in problem.properties: False\n",
+      "Adding AngularMomentum to problem.properties...\n",
+      "AngularMomentum is in problem.properties: True\n",
+      "Discarding AngularMomentum from problem.properties...\n",
+      "AngularMomentum is in problem.properties: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.properties import AngularMomentum\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)\n",
+    "\n",
+    "print(\"Adding AngularMomentum to problem.properties...\")\n",
+    "base_problem.properties.add(AngularMomentum(2))\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)\n",
+    "\n",
+    "print(\"Discarding AngularMomentum from problem.properties...\")\n",
+    "base_problem.properties.discard(AngularMomentum)\n",
+    "\n",
+    "print(\"AngularMomentum is in problem.properties:\", AngularMomentum in base_problem.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b72239b",
+   "metadata": {},
+   "source": [
+    "The specific subclasses of `BaseProblem` for the various stacks such as [electronic structure](01_electronic_structure.ipynb), [vibrational structure](02_vibrational_structure.ipynb) or [lattice models](10_lattice_models.ipynb) make things even simpler for you, because they provide direct attributes for the built-in properties of Qiskit Nature:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "6c251832",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<qiskit_nature.second_q.properties.particle_number.ParticleNumber object at 0x14bfa1aa5b80>\n",
+      "<qiskit_nature.second_q.properties.angular_momentum.AngularMomentum object at 0x14bfa1aa5a60>\n",
+      "<qiskit_nature.second_q.properties.magnetization.Magnetization object at 0x14bfa1aa5ac0>\n",
+      "<qiskit_nature.second_q.properties.dipole_moment.ElectronicDipoleMoment object at 0x14bfa1aa5f70>\n",
+      "None\n"
```

Comment:
This shows up as blank lines and None when I look at the output here. Ie the output of the source below.

I guess the None is expected though as it then says

>"You can see, that all properties come pre-populated except for the `ElectronicDensity`

Maybe this is not for this tutorial but the fields on the `properties` like `electronic_density` are they dynamic in that I can add some new property and it will have a field like this. Maybe this is more for some user defined property bit later.

Reply:
These fields are not dynamic. The tutorial states:

> The specific subclasses of BaseProblem for the various stacks such as <...> make things even simpler for you, because they provide direct attributes for the built-in properties of Qiskit Nature:

So the fields only exist for the built-in properties of Nature. User-provided ones need to be accessed via sets